vyper/parser/self_call.py

```diff
@@ -163,30 +185,42 @@ def call_self_private(stmt_expr, context, sig):
                         ['mstore', begin_pos, 'pass'],  # get len
                         ['mstore', i_placeholder, 0],
                         ['label', start_label],
-                        ['if', ['ge', ['mload', i_placeholder], ['ceil32', ['mload', begin_pos]]], ['goto', end_label]],  # break
-                        ['mstore', ['add', ['add', begin_pos, 32], ['mload', i_placeholder]], 'pass'],  # pop into correct memory slot.
-                        ['mstore', i_placeholder, ['add', 32, ['mload', i_placeholder]]],  # increment i
+                        [  # break
+                            'if',
+                            ['ge', ['mload', i_placeholder], ['ceil32', ['mload', begin_pos]]],
+                            ['goto', end_label]
+                        ],
+                        [  # pop into correct memory slot.
+                            'mstore',
+                            ['add', ['add', begin_pos, 32], ['mload', i_placeholder]],
+                            'pass',
+                        ],
+                        # increment i
+                        ['mstore', i_placeholder, ['add', 32, ['mload', i_placeholder]]],
                         ['goto', start_label],
                         ['label', end_label]],
                     typ=None, annotation='dynamic unpacker', pos=getpos(stmt_expr))
                 pop_return_values.append(o)
 
-    call_body = (
-        ['seq_unchecked'] +
-        pre_init +
-        push_local_vars +
-        push_args +
-        jump_to_func +
-        pop_return_values +
-        pop_local_vars +
-        [returner]
-    )
+    call_body = list(itertools.chain(
```

Comment:
Can you explain this change?

Reply:
Flake8 doesn't like binary operators hanging off the beginning or end of lines.  That replacement code is functionally equivalent to using `+` to concatenate lists.